src/libraries/Common/src/System/Security/Cryptography/MLDsaCng.Windows.cs

```diff
@@ -95,16 +95,21 @@ static Exception DebugFailAndGetException(string? parameterSet)
             }
         }
 
-        public partial CngKey Key
+        public partial CngKey GetKey()
         {
-            get
-            {
-                ThrowIfDisposed();
+            ThrowIfDisposed();
 
-                return _key;
-            }
+#if SYSTEM_SECURITY_CRYPTOGRAPHY
```

Comment:
I've seen this split compilation several times now regarding duplicating CngKey efficiently. I am going to clean this up in a separate PR, but for now, follow the same pattern.